plivo/resources/token.py

```diff
@@ -0,0 +1,35 @@
+import json
+from plivo.base import (ListResponseObject, PlivoResource,PlivoResourceInterface)
+from plivo.utils import to_param_dict
+from plivo.utils.validators import *
+
+class Token(PlivoResourceInterface):
+    @validate_args(
+        iss = [required(of_type(six.text_type))],
+        auth_token = [required(of_type(six.text_type))],
+        sub = [optional(of_type(six.text_type))],
+        nbf = [required(of_type(six.text_type))],
+        exp = [required(of_type(six.text_type))],
+        incoming_allowed = [optional(of_type(bool))],
+        outgoing_allowed = [optional(of_type(bool))],
+        app = [optional(of_type(six.text_type))]
+    )
+
+    def create( self,iss, auth_token, sub,nbf, exp, incoming_allowed, outgoing_allowed,app):
```

Comment:
Add validation check for "iss". It is a mandatory field and must be present

Reply:
Updates are added